partner_communication_switzerland/models/contracts.py

```diff
@@ -464,3 +465,16 @@ def _new_dossier(self):
             if sub.correspondant_id.id != sub.partner_id.id:
                 sub.send_communication(
                     sub_proposal_config, correspondent=False)
+
+
+class ContractGroup(models.Model):
+    _inherit = 'recurring.contract'
+
+    send_to_4m = fields.Date('Sent to 4M')
+
+
+class AccountInvoiceLine(models.Model):
+    _name = 'account.invoice.line'
+    _inherit = 'account.invoice.line'
+
+    send_to_4m = fields.Date('Sent to 4M')
```

Comment:
Put this in the file account_invoice_line.py where the object is already inherited.

Reply:
The name shoud be `sent_to_4m`